Homework_3/Task_3.py

```diff
@@ -0,0 +1,11 @@
+import csv
+import datetime
+import time
+
+with open('new.csv', 'w', encoding='utf-8') as file:
+    my_writer = csv.writer(file)
+    names_of_columns = ['№', 'Seconds', 'Milliseconds']
+    my_writer.writerow(names_of_columns)

```

Comment:
Не критично, но names_of_columns больше нигде не используются и они не сложные для прочтения(их немного), а потому их можно не записывать в переменную
```suggestion
my_writer.writerow(['№', 'Seconds', 'Milliseconds'])
```

Reply:
А в целом решение гуд)